client/app/core/tagging.service.js

```diff
@@ -1,5 +1,5 @@
 /** @ngInject */
-export function taggingService(CollectionsApi, lodash, exception) {
+export function TaggingService(CollectionsApi, lodash, exception) {
```

Comment:
Ah looks like it is used in one place: https://github.com/ManageIQ/manageiq-ui-service/pull/530/files#diff-73359eb751fda8d22574afaa8f29469cR81

Starting to wonder if there is some kind of linter we can set-up to catch these kinds of things.